gateway/webrtc/client.js

```diff
@@ -227,17 +244,11 @@ loginBtn.addEventListener("click", function(event) {
 function handleVideo(myStream) {
     console.log("Entering handleVideo", myStream);
     stream = myStream;
-
-
-    //displaying local video stream on the page
-    if (window.URL) {
-        localVideo.src = window.URL.createObjectURL(stream);
-    } else {
-        localVideo.src = stream;
-    }
+    localVideo.srcObject = stream;
```

Comment:
I have changed this because it is shorter and reduces the number of lines, thus increasing readability. Furthermore, https://www.w3.org/TR/webrtc/  uses only srcObject in examples and not src.